app/models/journeys/adjustments/WhichYearIsLossReported.scala

```diff
@@ -44,4 +52,14 @@ object WhichYearIsLossReported extends Enumerable.Implicits {
   implicit val enumerable: Enumerable[WhichYearIsLossReported] =
     Enumerable(values.map(v => v.toString -> v): _*)
 
+  private def generateYears(upTo: Int = 5): Seq[String] = {
```

Comment:
I would probably use hardcoded enums, and update it every year. But if it is not doable I would implement dynamic function and put it in our common place where we have year-dependent code.

This function will probably go, but just for future: this function is pretty complex, I would unit test it, especially edge cases. To do this you need to make `private[adjustments]` so it is open in the same package and easily tested